scripts/ci/download_devtools_regression_build.js

```diff
@@ -108,6 +109,22 @@ async function downloadRegressionBuild() {
       )} ${buildPath}`
     );
   }
+
+  if (semver.gte(reactVersion, '18.2.0') && semver.lt(reactVersion, '19')) {
+    console.log(chalk.white(`Downloading react-compiler-runtime\n`));
+    await exec(
+      `npm install --prefix ${REGRESSION_FOLDER} react-compiler-runtime`
+    );
+
+    console.log(
+      chalk.white(
+        `Moving react-compiler-runtime to react/compiler-runtime.js\n`
+      )
+    );
+    await exec(
+      `mv ${REGRESSION_FOLDER}/node_modules/react-compiler-runtime/dist/index.js ${buildPath}/react/compiler-runtime.js`
```

Comment:
It seems confusing and maybe unnecessary to overrwrite react/compiler-runtime.js, but I see why it's useful to

Reply:
> to overrwrite react/compiler-runtime.js

`react/compiler-runtime.js` is only shipped in React 19, and this check is gated for React [18.2, 19), so we don't override anything, just copying a script to replicate same `react` package structure